run_test.sh

```diff
@@ -143,6 +148,7 @@ function help {
   echo "-l              Run lint test"
   echo "-m              Run sample and generate main dc output"
   echo "-p              Run python tests"
+  echo "-protoc         Compile protos"
```

Comment:
should this be --protoc (otherwise could mean -p & -r & -o)

Reply:
Good point. Done.